### PR TITLE
sysvar: update doc for tidb_opt_prefer_range_scan

### PR DESCRIPTION
**Pull request by xuyifangreeneyes**
### What is changed, added or deleted? (Required)

Update doc for `tidb_opt_prefer_range_scan`.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): [tidb/pull/27123](https://github.com/pingcap/tidb/pull/27123)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1109,10 +1109,10 @@ mysql> desc select count(distinct a) from test.t;
 
 ### `tidb_opt_prefer_range_scan` <span class="version-mark">从 v5.0 版本开始引入</span>
 
-- 作用域：SESSION
+- 作用域：SESSION | GLOBAL
 - 默认值：`OFF`
```
Comment (TomShawn) [inline system-variables.md:1113]: 1/0，ON/OFF 是否需要统一yixiane1？另外，这个属于 5.2 有的内容吗？

Comment (xuyifangreeneyes) [reply system-variables.md:1113]: 不属于 5.2 的内容，要不统一成 ON/OFF ?

Comment (TomShawn) [reply system-variables.md:1113]: 好的

**Review by winoros — APPROVED** [review]

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**